etna/models/nn/chronos/chronos.py

```diff
@@ -0,0 +1,167 @@
+from pathlib import Path
+from typing import List
+from typing import Sequence
+
+from etna import SETTINGS
+from etna.datasets import TSDataset
+from etna.models.nn.chronos.base import ChronosBaseModel
+
+if SETTINGS.chronos_required:
+    import torch
+
+_DOWNLOAD_PATH = Path.home() / ".etna" / "chronos-models" / "chronos"
+
+
+class ChronosModel(ChronosBaseModel):
+    """
+    Class for pretrained chronos models.
+
+    This model is only for zero-shot forecasting: it doesn't support training on data during ``fit``.
+
+    Methods ``save`` and ``load`` do nothing.
```

Comment:
Why?

Reply:
Absence of these methods could break some logic of user's code. 
For example, he has a code that saves the model and loads it later for inference. Before this model this logic worked for any etna model but there should be some kind of hard-coded version only for this model.

Reply:
Ok
Let's `save` will save only class attributes without weights
`load` will call `__init__` with loaded parameters and `__init__` will load load weights.

Reply:
Seems fine. But we should describe such behavior in the docstring of `save` and `load` methods.